### PR TITLE
feat(blog): add extra information field for article

### PR DESCRIPTION
**Pull request by a-sava**
Extra data field uses to store some information about article, that can differ in different applications, for example in one need to store some extra links, in another extra authors and so on

### CODE REVIEW

File: microservices/blog/src/entities/article.ts [hunk 1]
```diff
@@ -54,6 +54,16 @@ class Article {
...
 
+  @JSONSchema({
+    description:
+      'Set of key-value pairs that can be attach to an article.' +
+      'This can be useful for storing extra information about the article in a structured format.',
+  })
+  @Column({ type: 'json', default: {} })
+  @IsUndefinable()
+  @IsObject()
+  extra: Record<string, any>;
```
Comment (OlegDO) [inline microservices/blog/src/entities/article.ts:71]:
```suggestion
  params: IParams;
```

File: microservices/blog/migrations/1693301991584-init.ts [hunk 2]
```diff
@@ -8,7 +8,7 @@ export default class init1693301991584 implements MigrationInterface {
      * Tables
      */
     await queryRunner.query(
-      `CREATE TABLE "article" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "userId" character varying(36), "title" character varying(255) NOT NULL, "alias" character varying(255) NOT NULL, "description" character varying(255) NOT NULL, "content" text NOT NULL DEFAULT '', "publishDate" TIMESTAMP, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "article(uq):alias" UNIQUE ("alias"), CONSTRAINT "article(pk):id" PRIMARY KEY ("id"))`,
+      `CREATE TABLE "article" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "userId" character varying(36), "title" character varying(255) NOT NULL, "alias" character varying(255) NOT NULL, "description" character varying(255) NOT NULL, "content" text NOT NULL DEFAULT '', "publishDate" TIMESTAMP,  "extra" json NOT NULL DEFAULT '{}', "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "article(uq):alias" UNIQUE ("alias"), CONSTRAINT "article(pk):id" PRIMARY KEY ("id"))`,
```
Comment (OlegDO) [inline microservices/blog/migrations/1693301991584-init.ts:11]: Should be params instead of extra

File: microservices/blog/__mocks__/article.ts [hunk 3]
```diff
@@ -5,6 +5,9 @@ const articleMock = {
     'In by an appetite no humoured returned informed. Possession so comparison inquietude he he conviction no decisively.',
   content:
     '<div><p>Marianne jointure attended she hastened surprise but she.</p><p> Ever lady son yet you very paid form away.</p></div>',
+  extra: {
+    links: ['https://google.com', 'https://wikipedia.com'],
```
Comment (OlegDO) [inline microservices/blog/__mocks__/article.ts:9]: Author name should be retrieved by  userId

**Review by OlegDO — CHANGES_REQUESTED** [review]

File: microservices/blog/src/entities/article.ts (hunk 1 above)
Comment (a-sava) [reply microservices/blog/src/entities/article.ts:71]: information can differ in different applications

File: microservices/blog/migrations/1693301991584-init.ts (hunk 2 above)
Comment (a-sava) [reply microservices/blog/migrations/1693301991584-init.ts:11]: it's not params field, that field uses to store some extra data

File: microservices/blog/src/entities/article.ts [hunk 4]
```diff
@@ -54,6 +54,16 @@ class Article {
   @IsNullable()
   publishDate: string | null;
 
+  @JSONSchema({
+    description:
+      'Set of key-value pairs that can be attach to an article.' +
```
Comment (OlegDO) [inline microservices/blog/src/entities/article.ts]: Useless description

**Review by OlegDO — APPROVED** [review]

**Pull request merged**